econf/Cargo.toml

```diff
@@ -16,6 +16,7 @@ serde = "1.0"
 serde_yaml = "0.9"
 econf-derive = { version = "0.3.0", path = "../econf-derive" }
 humantime = "2.1"
+url = "2.5"
```

Comment:
I wonder if we should have a feature flag for third party types 🤔

Reply:
Done! https://github.com/YushiOMOTE/econf/compare/27020801235d76b6a33c8e83a8827a82788a8fca..4b26e973b22a40236cbcc26f42f24e162224aca4

Running `cargo test`:
<img width="1639" height="1863" alt="image" src="https://github.com/user-attachments/assets/348c3a8a-64fb-4fe3-8a60-f91d5b912471" />

And running with all feature flags `cargo test --all-features`:
<img width="1639" height="1950" alt="image" src="https://github.com/user-attachments/assets/5c2576ad-37df-4b24-935d-fcbe3965a7dd" />